include/config/config_rom.h

```diff
@@ -14,7 +14,7 @@
  * Enables Rumble Pak Support.
  * Currently not recommended, as it may cause random crashes.
  */
-// #define ENABLE_RUMBLE (1 || VERSION_SH)
+#define ENABLE_RUMBLE
```

Comment:
Either this should be disabled by default, or if it is safe to always leave on, the comment two lines up should be modified/deleted.

Reply:
I removed the comment. I think it should be on by default since it's never crashed in any of my testing with these changes. I think that comment was about the old PJ64 crash from around 2 years ago.